packages/core/src/layout/BottomLink/BottomLink.tsx

```diff
@@ -50,15 +53,15 @@ const BottomLink: FC<Props> = ({ link, title, onClick }) => {
   return (
     <div>
       <Divider />
-      <Link href={link} onClick={onClick}>
+      <Link href={link} onClick={onClick} style={{ textDecoration: 'none' }}>
```

Comment:
For some reason doing something like `classes.link` does not work here 🤔

Reply:
Check what the docs say

https://material-ui.com/components/links/

and also 

https://material-ui.com/api/link/

We are supposed to use limited styling on these since they should be "framework controlled" so to speak. But there is a specific `underline="none"` prop for this use case.

Reply:
Fixed.